core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -125,8 +130,8 @@ protected void inboundHeadersReceived(Metadata.Headers headers) {
    *
    * @param frame the received data frame. Its ownership is transferred to this method.
    */
-  protected void inboundDataReceived(ReadableBuffer frame) {
-    Preconditions.checkNotNull(frame, "frame");
```

Comment:
@nmittler, then after this PR gets checked in I'll send you a PR reverting the `checkNotNull` changes? I'll send out a cleanup PR that changes all usages of `checkNotNull` to be the non-static version.

I will note that we already have this issue today. This is why I keep on pointing on import order issues, because people have misconfigured IDEs and they keep fighting with each other. This is also why I point out other random reformatting changes that aren't necessary because I do know that people's IDEs are fighting for the formatting, and it does cause constant noise and merge conflicts, and people should fix their IDEs, deal with the annoyance of having to revert such changes, or stop using IDE features that are causing the trouble.